scene/3d/cpu_particles_3d.cpp

```diff
@@ -792,6 +792,13 @@ void CPUParticles3D::_particles_process(double p_delta) {
 					//do none
 				} break;
 				case EMISSION_SHAPE_SPHERE: {
+					real_t s = 2.0 * Math::randf() - 1.0;
+					real_t t = Math_TAU * Math::randf();
+					real_t x = Math::randf();
+					real_t radius = emission_sphere_radius * Math::sqrt(1.0 - s * s);
+					p.transform.origin = Vector3(Math::lerp(0, radius * Math::cos(t), x), Math::lerp(0, radius * Math::sin(t), x), Math::lerp(0, emission_sphere_radius * s, x));
```

Comment:
I think using ``Vector3.lerp`` is a little easier to read. IMO it makes it more obvious that we are interpolating between the sphere origin and the surface of the sphere by random value ``x``
```suggestion
					p.transform.origin = Vector3(0,0,0).lerp(Vector3(radius * Math::cos(t), radius * Math::sin(t), emission_sphere_radius * s), x);
```

Reply:
Thanks for your feedback, the suggested code is much easier to read!